kornia/feature/defmo.py

```diff
@@ -0,0 +1,112 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+import torchvision.models
+
+urls: Dict[str, str] = {}
+urls["defmo_encoder"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/encoder_best.pt"
+urls["defmo_rendering"] = "http://ptak.felk.cvut.cz/personal/rozumden/defmo_saved_models/rendering_best.pt"
+
+
+class EncoderDeFMO(nn.Module):
+    def __init__(self):
+        super(EncoderDeFMO, self).__init__()
+        model = torchvision.models.resnet50(pretrained=True)
+        modelc1 = nn.Sequential(*list(model.children())[:3])
+        modelc2 = nn.Sequential(*list(model.children())[4:8])
+        pretrained_weights = modelc1[0].weight
```

Comment:
Is the pretrained weight not stored in the encoder_best.pt?

Reply:
It's initialization for "training". But you're right that it's not needed for the inference. Should I remove it?

Reply:
Removed.